tools/test/examples/examples_lib.py

```diff
@@ -484,17 +493,25 @@ def symlink_mbedos(config, path, exp_filter):
             else:
                 logging.info("Creating Symbolic link '%s'->'mbed-os'" % path)
                 os.symlink(path, "mbed-os")
+                open('mbed-os.lib', 'a').close()
```

Comment:
This probably needs a comment explaining why it is here. It may not be needed in the current version (or future versions) of mbed-tools.

Reply:
comment added